doc/specs/stdlib_stringlist.md

```diff
@@ -0,0 +1,467 @@
+in any position without needing to fill in the intervening positions.
+
+#### Syntax
+
+`length = list % [[stringlist_type(type):length(bound) ()]]`
+
+#### Class
+
+Function
+
+#### Arguments
+
+`list`: the stringlist variable to retrieve the length from
+
+#### Result value
+
+Returns the highest index of a string that has been set.
+
+
+
+### `range` - retrieve a range of strings from the list
+
+#### Status
+
+Experimental
+
+#### Description
+
+Retrieve the strings occurring between the given positions as a new list.
+
+#### Syntax
+
+`rangelist = list % [[stringlist_type(type):range(bound) ( first, last )]]`
+
+#### Class
+
+Function
+
+#### Arguments
+
+`list`: the stringlist variable to insert the string(s) into
+
+`first`: the position of the first string to be retrieved
+
+`last`: the position of the last string to be retrieved
+
+#### Result value
+
+The result is a new list containing all the strings that appear from the first to the last position, inclusively.
+
+
+
+### `replace` - replace one or more strings between two given positions
+
+#### Status
+
+Experimental
+
+#### Description
+
+Replace one or more strings between two given positions. The new strings may be given as a single string, a list of
+strings or a plain array.
+
+#### Syntax
+
+`call list % [[stringlist_type(type):replace(bound) ( first, last, string )]]`
+
+#### Class
+
+Subroutine
+
+#### Arguments
+
+`list`: the stringlist variable to replace the string(s) in
+
+
+`first`: the position of the first string to be retrieved
+
+`last`: the position of the last string to be retrieved
+
+`string`: the string to be inserted, a list of strings or a plain array of strings
+
+
+
+### `sort` - return a sorted list
+
+#### Status
+
+Experimental
+
+#### Description
+
+Create a new list consisting of the sorted strings of the given list. The strings are sorted according to ASCII, either
+in ascending order or descending order.
+
+#### Syntax
+
+`sortedlist = list % [[stringlist_type(type):sort(bound) ( ascending )]]`
+
+#### Class
+
+Subroutine
+
+#### Arguments
+
+`list`: the stringlist variable of which the contents should be copied
+
+`ascending` (optional): if not present or true, sort the list in ascending order, otherwise descending
+
+#### Result value
+
+The contents of the given list is sorted and then stored in the new list.
+
+
+### `=` - copy the contents of a list
+
+#### Status
+
+Experimental
+
+#### Description
+
+Copy an existing list to a new one. The original list remains unchanged.
+
+#### Syntax
+
+`copylist = list`
+
+#### Class
+
+Assignment
+
+#### Operands
+
+`list`: the stringlist variable to be copied
+
+
+
+### `//` - concatenate a list with one or more strings
+
+#### Status
+
+Experimental
+
+#### Description
+
+Concatenate a list with a string, a list of strings or a plain array
+
+#### Syntax
+
+`concatenatedlist = list // string`
+
+`concatenatedlist = string // list`
+
+#### Class
+
+Assignment
```

Comment:
```suggestion
Operator
```

Reply:
The standard makes a difference between assignment and operator ... I can live with either, though.